files/usr/local/bin/clean_zombie.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-<<'LICENSE'
+true <<'LICENSE'
```

Comment:
what is "true" doing?

Reply:
This resolved a shellcheck warning I ran into.  This is a shell pipe command going into nothing.